### PR TITLE
Adds support of pipeRDD().

### PR DESCRIPTION
**Pull request by hlin09**

### CODE REVIEW

File: pkg/R/RDD.R [hunk 1]
```diff
@@ -1275,6 +1275,43 @@ setMethod("aggregateRDD",
             Reduce(combOp, partitionList, zeroValue)
           })
 
+#' Pipes elements to a forked externel process.
```
Comment (shivaram) [inline pkg/R/RDD.R]: `external` spelling error

File: pkg/R/RDD.R [hunk 2]
```diff
@@ -1275,6 +1275,43 @@ setMethod("aggregateRDD",
...
+  standardGeneric("pipeRDD") 
+})
+
+#' @rdname pipeRDD
+#' @aliases pipeRDD,RDD,character-method
+setMethod("pipeRDD",
+          signature(rdd = "RDD", command = "character"),
+          function(rdd, command, env = list()) {
+            func <- function(part) {
+              trim.trailing.func <- function(x) {
```
Comment (shivaram) [inline pkg/R/RDD.R]: It'll be good to add a test case where the `trim.trailing.func` is used

**Comment by shivaram** [comment]
Thanks @hlin09 for updating this PR. The change looks pretty good to me. I had a couple of minor comments inline

File: pkg/R/RDD.R (hunk 2 above)
Comment (hlin09) [inline pkg/R/RDD.R]: Done.

File: pkg/R/RDD.R (hunk 1 above)
Comment (hlin09) [inline pkg/R/RDD.R]: Thanks for the catch. Done.

**Comment by shivaram** [comment]
Thanks. LGTM - Merging this

**Pull request merged**